src/sentry/web/urls.py

```diff
@@ -364,6 +364,8 @@ def init_all_applications():
     url(r'^out/$', OutView.as_view()),
 
     url(r'^accept-transfer/$', react_page_view, name='sentry-accept-project-transfer'),
+    url(r'^settings/account/security', generic_react_page_view, name='sentry-account-settings-security'),
```

Comment:
Shouldn't the route here be a bit more specific? e.g. `r'^settings/account/security/$'`